cmd/config/docs/api-conventions/config-fn.md

```diff
@@ -222,15 +222,15 @@ Following is an example of running the `kustomize config run` using the precedin
     metadata:
       name: my-instance
       annotations:
+        config.k8s.io/function: |
+          container:
+            image: gcr.io/example-functions/nginx-template:v1.0.0
         config.kubernetes.io/local-config: "true"
-      configFn:
-        container:
-          image: gcr.io/example-functions/nginx-template:v1.0.0
     spec:
       replicas: 5
 
-  - `configFn.container.image`: the image to use for this API
-  - `annotations[config.kubernetes.io/local-config]`: mark this as not a Resource that should
+  - `annotations.[config.k8s.io/function]`: the image to use for this API
```

Comment:
we should tweak the messaging since this isn't the image anymore.  `the specification of how to run the function`?

Reply:
Tweaked :D